.pre-commit-config.yaml

```diff
@@ -21,7 +21,13 @@ repos:
       - id: black
         language_version: python3
   - repo: https://gitlab.com/pycqa/flake8
-    rev: 3.7.9
+    rev: 3.8.3
     hooks:
       - id: flake8
         language_version: python3
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.782
+    hooks:
+      - id: mypy
+        args: ["--strict", "--show-error-codes"]
+        additional_dependencies: ["numpy", "xarray", "pysnptools>=0.4.20", "git+https://github.com/pystatgen/sgkit"]
```

Comment:
`git+https://github.com/pystatgen/sgkit` oh god, that's a shame :(

linking https://github.com/pystatgen/sgkit/issues/65

Reply:
hahah yea not sure what to do about that